src/Advertising.js

```diff
@@ -21,26 +32,20 @@ export default class Advertising {
     }
   }
 
+
   // ---------- PUBLIC METHODS ----------
 
   async setup() {
     this.isPrebidUsed =
       typeof this.config.usePrebid === 'undefined' ? typeof window.pbjs !== 'undefined' : this.config.usePrebid
     this.isAPSUsed =
       typeof this.config.useAPS === 'undefined' ? typeof window.apstag !== 'undefined' : this.config.useAPS
-    this.executePlugins('setup')
+    this.executePlugins(EXECUTE_PLUGINS_ACTION.SETUP)
```

Comment:
This seems to break in the SSR. Could you please open the playground and test it out?

Reply:
@DanSnow 
I made the EXECUTE_PLUGINS_ACTION as a class member, then tested it on the playground, the console shows a consistent result for both the main branch and my branch on my side now.